src/slack.coffee

```diff
@@ -203,6 +205,14 @@ class SlackBot extends Adapter
     for msg in messages
       continue if msg.length < SlackBot.MIN_MESSAGE_LENGTH
 
+      # Replace @username with <@UXXXXX> for mentioning users
+      msg = msg.replace /(?:^@| @)([A-z]+)/gm, (match, p1) ->
```

Comment:
you should be able to use a "fat arrow" in here to avoid the `client = @client` part

http://coffeescript.org/#fat-arrow